### PR TITLE
Range fixes

### PR DESCRIPTION
**Pull request by abeaumont**

### CODE REVIEW

File: sources/dylan/apple-dylan-test-suite/test-range.dylan [hunk 1]
```diff
@@ -50,11 +50,11 @@ end test range-4;
 // not except through and up-to in a single expression
 //
 define test range-5 (description: "Through:, by:, and size:")
```
Comment (waywardmonkeys) [inline sources/dylan/apple-dylan-test-suite/test-range.dylan]: This used to test ``through:`` but now tests ``to:``, right?

So it probably makes sense to:

1. Test these error conditions.
2. Make sure that the correct cases are tested as well and return the correct set of values.

**Comment by waywardmonkeys** [comment]
The first commit has a typo in the commit message, "handlign", twice.

File: sources/dylan/apple-dylan-test-suite/test-range.dylan (hunk 1 above)
Comment (abeaumont) [inline sources/dylan/apple-dylan-test-suite/test-range.dylan]: That's what that description implies, but there's no such ``through:`` keyword anymore. I'll update the description and remove the comments above the test too, which don't apply anymore.

I'll add also correct cases too

**Comment by abeaumont** [comment]
Fixed the commit message typo

File: sources/dylan/apple-dylan-test-suite/test-range.dylan (hunk 1 above)
Comment (abeaumont) [inline sources/dylan/apple-dylan-test-suite/test-range.dylan]: Done

**Comment by waywardmonkeys** [comment]
Thanks!

**Pull request merged**